### PR TITLE
feature/clientssl profile renegotiation

### PR DESCRIPTION
**Pull request by lopf**

### CODE REVIEW

File: library/modules/bigip_profile_client_ssl.py [hunk 1]
```diff
@@ -173,6 +173,12 @@
       - once
       - always
     version_added: 2.8
+  renegotiation:
+    description:
+      - Enables or disables SSL renegotiation.  The default value is enabled.
```
Comment (wojtek0806) [inline library/modules/bigip_profile_client_ssl.py]: no need to specify the default value here just use default: yes, if this is referring to system default variable there is no need to provide this information. 

Just mention that this value is inherited from parent profile upon creation

File: library/modules/bigip_profile_client_ssl.py [hunk 2]
```diff
@@ -173,6 +173,12 @@
...
+        When creating a new profile, the setting is provided by the parent profile.
```
Comment (wojtek0806) [inline library/modules/bigip_profile_client_ssl.py]: add - to this to make it a new line

File: library/modules/bigip_profile_client_ssl.py [hunk 3]
```diff
@@ -666,6 +690,14 @@ def strict_resume(self):
             return 'yes'
         return 'no'
 
+    @property
+    def renegotiation(self):
+        if self._values['renegotiation'] is None:
```
Comment (wojtek0806) [inline library/modules/bigip_profile_client_ssl.py]: remove this this function already covers this:

https://github.com/F5Networks/f5-ansible/blob/devel/library/module_utils/network/f5/common.py#L222

just return result of flatten boolean

**Comment by wojtek0806** [comment]
@lopf 
Apart from changes since this is the first time I see your PR before I merge anything I need you to follow this please:

https://clouddocs.f5.com/products/orchestration/ansible/devel/development/cla-landing.html

**Comment by Cajga** [comment]
@lopf, @wojtek0806,
I was about to submit the same PR (although I would have missed some change). Would be nice to get this merged.
Any help required?

**Comment by Cajga** [comment]
@lopf,

You probably want to include your test into main.yaml like wojtek0806 did [here](https://github.com/F5Networks/f5-ansible/pull/1213/commits/77d6ad17e54363dbe399e0e293de6abdb706925d?utf8=%E2%9C%93&diff=split#diff-e6613dcb4f81972c1b8e21602a1c4d6d)

**Comment by lopf** [comment]
sure, thx, I added it.

**Comment by wojtek0806** [comment]
@lopf 
please squash these changes and i will re review as there are few things that need to be done

**Comment by wojtek0806** [comment]
also to get ahead of my comments, please open an issue for this feature, i will explain why during the review process

**Comment by wojtek0806** [comment]
looks good now

**Pull request merged**